integration-tests/src/test/java/io/datanerds/grpc/PingPongServer.java

```diff
@@ -0,0 +1,63 @@
+package io.datanerds.grpc;
+
+import io.datanerds.grpc.integrationtests.PingPongGrpc;
+import io.datanerds.grpc.integrationtests.PingPongProto;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class PingPongServer {
+    private static final Logger logger = LoggerFactory.getLogger(PingPongServer.class);
+    private Server server;
+    private final int port;
+
+    public PingPongServer() {
+        this(55551);
```

Comment:
Probably better use a random value here to decrease possibility of already taken ports?